### DISCUSSION TITLE
Nonadmin/VIP commands

### DISCUSSION THREAD

**Discussion by Dystroyertheguy** [Epix-Incorporated/Adonis#726] [General]
How do I edit adonis to be allow non-VIP to use !hat/!shirt/!pants commands?

**Comment by r-kinesis** [comment]
You need to check your Main Module for that.

**Comment by Expertcoderz** [comment]
A bit late for an answer, but anyways, and for those who may need it:

```lua
--// ModuleScript named "Server-DonorCommandUnrestrictor" (without quotes)
--// placed under Adonis_Loader > Config > Plugins
server, service = nil, nil
return function()
    for cmdName, cmd in pairs(server.Commands) do
        if cmd.AdminLevel == "Donors" then
            cmd.AdminLevel = "Players"
        end
    end
end
```

***Note: Don't touch the MainModule for this.* Loader plugins exist for a reason.**

**Reply by Dystroyertheguy** [reply]
Real quick, do you know how to fix the thing that when you run !removehats, you can't get the previous hats back?

**Reply by Expertcoderz** [reply]
> Real quick, do you know how to fix the thing that when you run !removehats, you can't get the previous hats back?

I'd suggest that you [submit an issue](https://github.com/Sceleratis/Adonis/issues/new/choose) so we can look into it in greater depth.
